Create Writing Day project FAQ

rosewms opened the pull request:
Based on the content from the WTD <3's OSS blog: https://www.writethedocs.org/conf/portland/2023/news/writing-day-announcement/

<!-- readthedocs-preview writethedocs-www start -->
----
:books: Documentation preview :books:: https://writethedocs-www--1946.org.readthedocs.build/

<!-- readthedocs-preview writethedocs-www end -->

rosewms committed:
Create Writing Day project FAQ

Based on the content from the WTD <3's OSS blog: https://www.writethedocs.org/conf/portland/2023/news/writing-day-announcement/

rosewms committed:
Add WD Project FAQ link to Writing Day event

rosewms committed:
Spelling fix

rosewms committed:
Create writing-day-project-faq.rst

Add placeholder doc

plaindocs commented on docs/include/conf/virtual/writing-day-project-faq.rst:
```diff
@@ -0,0 +1,129 @@
+
+**Reminder**: Documentarians have a variety of backgrounds and may need additional info to be successful in onboarding to your project!
+
+.. _writing-day-case:
```
```suggestion
.. _making-writing-day-case:
```

plaindocs commented on docs/include/conf/virtual/writing-day-project-faq.rst:
```diff
@@ -0,0 +1,129 @@
+they are to attend to learn more. It benefits us, them, and you!
+
+Need help convincing your OSS project community or corporate benefactors on the benefits of attending Writing Day? 
+See :ref:`writing-day-case`.
```
```suggestion
See :ref:`making-writing-day-case`.
```

plaindocs committed:
Apply suggestions from code review

plaindocs commented on docs/include/conf/virtual/writing-day-project-faq.rst:
```diff
@@ -0,0 +1,129 @@
+Submit your project
+^^^^^^^^^^^^^^^^^^^
+
+There is a basic level of information we need to add your project to the Writing Day list. `Submitting the form <>`_ is an
```
@rosewms would you mind creating and linking the form here so the build doesn't break?

rosewms committed:
Added form link

I recreated the form in the Atlantic 2023 drive, I did not have the permissions to move a copy or the Portland 2023 form.

rosewms committed:
Merge branch 'main' into writing-day-project-faq
